Dockerfile.base

```diff
@@ -1,4 +1,4 @@
-FROM node:6.9.1-alpine
+FROM node:0.0.0-alpine
```

Comment:
These are templates. The whole point of this PR is that those `0.0.0` are replaced by the version you want to release by the script